src/splitwisesdk/core/Group.java

```diff
@@ -0,0 +1,20 @@
+package splitwisesdk.core;
+
+import java.util.ArrayList;
+
+public class Group {
+
+    public String id;
+    public ArrayList<GroupMember> members = new  ArrayList<>();
+    public ArrayList<Expense> expenses = new ArrayList<>();
+
+
+
```

Comment:
add method to boolean removeGroupMember(GroupMember groupMember) {
      // Verify that groupMember is in members
      // If yes then remove it and return true
      // else do nothing and return false
}